src/Exports/DonorsExport.php

```diff
@@ -26,6 +26,11 @@ class DonorsExport extends Give_Batch_Export
     /** @var String */
     protected $searchBy;
 
+    /**
+     * @var int
+     */
+    protected $donationForm;
```

Comment:
Can we rename this to be more descriptive?

```suggestion
    protected $donationFormId;
```

Reply:
Resolved https://github.com/impress-org/givewp/pull/7146/commits/d99d6cf5e7ef235e94909da13b1792212bb1722a